File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/items/files/CheckForAnyStoredFiles.java
package com.lasthopesoftware.bluewater.client.stored.library.items.files;

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId;
import com.namehillsoftware.handoff.promises.Promise;

public interface CheckForAnyStoredFiles {
	Promise<Boolean> promiseIsAnyStoredFiles(LibraryId libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/items/files/CheckForAnyStoredFiles.kt
package com.lasthopesoftware.bluewater.client.stored.library.items.files

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId
import com.namehillsoftware.handoff.promises.Promise

interface CheckForAnyStoredFiles {
    fun promiseIsAnyStoredFiles(libraryId: LibraryId): Promise<Boolean>
}
